src/clusterfuzz/_internal/cron/triage.py

```diff
@@ -261,6 +261,7 @@ def _file_issue(testcase, issue_tracker, throttler):
 
   if throttler.should_throttle(testcase):
     _add_triage_message(testcase, 'Skipping filing as it is throttled.')
+    monitoring_metrics.ISSUE_FILING_THROTTLED.increment({'fuzzer_name': testcase.fuzzer_name})
```

Comment:
One alternative would be to have a single ISSUE_FILLING metric, and a label to indicate throttled/success/failure, to define less metric types.

Reply:
I think it might be more convenient to have a single issue_filing metric with 3 possible values (success, failure, throttling). Same for ISSUE_CLOSING which can have two values(success, failure)

Reply:
Will send a followup one to simplify this.